Content.Server/Body/Systems/MetabolizerSystem.cs

```diff
@@ -163,7 +170,7 @@ private void TryMetabolize(EntityUid uid, MetabolizerComponent? meta=null, Mecha
 
                         if (effect.ShouldLog)
                         {
-                            var entity = args.SolutionEntity;
+                            var entity = bodyEntityUid != null ? bodyEntityUid.Value : args.SolutionEntity;
```

Comment:
You can just replace this with `actualEntity`.

Also instead of `bodyEntityUid != null ? bodyEntityUid.Value : args.SolutionEntity` maybe just do `bodyEntityUid ??  args.SolutionEntity`